Makefile

```
@@ -0,0 +1,147 @@
+.PHONY: help build clean test test-unit test-integration test-business \
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)